docs/contributor/data_representations.md

```diff
@@ -228,3 +228,17 @@ integer denotes the denominator of the represented fraction.
 ║ what ║ value tag │ boxed number tag │ numerator (i32) │ denominator (u32)  ║
```

Comment:
Maybe remove `(i32)` and `(u32)` here and leave just `│ numerator │ denominator  ║` ?

Reply:
Good catch!